e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -391,8 +391,8 @@ $deploymentName = "IotE2eInfra-$randomSuffix"
 # Deploy
 Write-Host @"
     `nStarting deployment which may take a while
-    1.Progress can be monitored from the Azure Portal (http://portal.azure.com)
-    2.Deployment name ($deploymentName), Resource group ($ResourceGroup), Subscription ($SubscriptionId)
+    1. Progress can be monitored from the Azure Portal (http://portal.azure.com).
+    2. Info to track: deployment name ($deploymentName), resource group ($ResourceGroup), subscription ($SubscriptionId).
```

Comment:
Should we specify that in order to track this deployment, go to resource group -> deployments -> deployment name.

It took me a while time to locate the deployment progress since I was searching on Azure by deployment name, which returns 0 results.